src/mock-builders/api/query.js

```diff
@@ -0,0 +1,20 @@
+import { mockedApiResponse } from './utils.js';
+
+/**
+ * Returns the api response for query api
+ *
+ * api - /query
+ *
+ * @param {*} messages Array of message objects.
+ */
+export const queryApi = (channel, messages = []) => {
```

Comment:
We already have this one - getOrCreateChannelApi

Reply:
Ah. I think I'm not quite familiar enough with how our APIs are structured yet, and didn't realise it was the same thing (because to me the name implies that this just gets the channel itself, but not its messages). Will replace :)

Reply:
@vishalnarkhede done